src/main/java/StringManipulation.java

```diff
@@ -1,33 +1,115 @@
+//Author: Jonathan Jarman
+//Date: 6/11/2023
+//File: String Manipulation
+/*The StringManipulation class provides a set of methods for manipulating strings,
+such as counting the number of words, removing characters at specific positions,
+extracting substrings, and restoring shuffled strings. It implements the StringManipulationInterface,
+which defines the contract for these operations.
+* */
 public class StringManipulation implements StringManipulationInterface {
 
+    String string; // holding string input
```

Comment:
can be changed to private for better encapsulation

Reply:
Added private modifier